### ISSUE TITLE
FEAT: Select next occurence of text selection

### ISSUE THREAD

**Issue by DKhalil** [beekeeper-studio/beekeeper-studio#900]
When editing larger queries it would be immensely helpful to have the possibility to select the next occurrence of a currently selected text snippet, like in VSCode (CTRL + D)

**Comment by rathboma** [comment]
Yeah there are a lot of text editor 'things' that beekeeper needs to improve on. Agreed on this one
